src/CMake/icd.cmake

```diff
@@ -6,7 +6,7 @@ SET (ICD_FILE_NAME "xilinx.icd")
 message("-- Preparing OpenCL ICD ${ICD_FILE_NAME}")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`